### PR TITLE
Here I come correcting people like I usually do :)

### PR DESCRIPTION
**Pull request by duckyondiscord**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -7,30 +7,28 @@
   <a href="https://discord.gg/3CVqX5bphG" alt="Discord"><img src="https://img.shields.io/discord/829309363213565982?label=Discord&style=flat&logo=discord&logoColor=white"></img></a>
   <a href="https://github.com/raluvy95/jolly/issues" alt="Issues"><img src="https://img.shields.io/github/issues/raluvy95/jolly"></img></a>
 </p>
-A discord bot designed for one server at once (private bot). Rewritten in
-TypeScript using Deno. The predecessor variant of this can be found in
+A discord bot designed for my server (private bot). Rewritten in
+TypeScript using Deno. Its predecessor can be found in
```
Comment (LiveEmily) [inline readme.md:11]: It's

File: readme.md [hunk 2]
```diff
@@ -7,30 +7,28 @@
...
+repository `git clone https://github.com/raluvy95/jolly.git`<br> **Step 2**:
+Rename `config-temp.json` to `config.json`<br> **Step 3**:
+Open a terminal<br> **Step 4**: Run the bot with `deno task run`
 
 # Advantages of Jolly over Maffin
 
-- Jolly is rewritten in TypeScript which takes seriously when it comes to
-  runtime error. So no more unexpected crash and no more TypeError in random
-  cases.
+- Jolly is rewritten in TypeScript which takes fixing runtime errors more seriously. So no more unexpected crash and no more random TypeErrors
```
Comment (LiveEmily) [inline readme.md:29]: crashes

File: readme.md [hunk 3]
```diff
@@ -7,30 +7,28 @@
...
 - No more insane breaking changes from Discord API library
-- Have own command handler, embed, utils and more
+- Has its own command handler, embed, utils and more
```
Comment (LiveEmily) [inline readme.md:31]: it's, embeds

File: readme.md (hunk 2 above)
Comment (duckyondiscord) [reply readme.md:29]: You're right about that one

File: readme.md (hunk 3 above)
Comment (duckyondiscord) [reply readme.md:31]: no, pretty sure it's "its"

File: readme.md (hunk 1 above)
Comment (duckyondiscord) [reply readme.md:11]: I'm pretty sure it's still "its", but feel free to look it up somewhere idk

Comment (LiveEmily) [reply readme.md:11]: ~~it's is an abbreviation of it is, making it it's, just like don't and can't being abbreviations of do not and can not~~
it's possessive so it is correct lmao

File: readme.md [hunk 4]
```diff
@@ -7,30 +7,28 @@
...
 </p>
-A discord bot designed for one server at once (private bot). Rewritten in
-TypeScript using Deno. The predecessor variant of this can be found in
+A discord bot designed for my server (private bot). Rewritten in
+TypeScript using Deno. Its predecessor can be found in
 <a href="https://github.com/raluvy95/maffin">another repository</a>
 
 You can contribute whatever you want to improve the bot with additional features
-and fixed bugs.
+bug fixes.
```
Comment (LiveEmily) [inline readme.md:15]: ``and bug fixes`` would have probably been better but this is just a minor remark and might just be an oopsie

**Pull request merged**